cumulus_library_umls/ancilary_tables.sql

```diff
@@ -0,0 +1,349 @@
+-- A collection of convenience views for UMLS data
+
+-- Selecting only child relationships FROM mrrel
+CREATE TABLE IF NOT EXISTS umls__mrrel_is_a AS
```

Comment:
This `umls__mrrel_is_a` only applies to DRUGS, recommend changing name to 
`umls__mrrel_drug_is_a`

Reply:
ok, done